File: Cpp/Find Count of Most Frequent Item in an Array.cpp
// https://www.codewars.com/kata/56582133c932d8239900002e/train/cpp

#include <bits/stdc++.h>
#include <vector>
using namespace std;

unsigned int most_frequent_item_count(const vector<int>& collection) {
    int result = 0;

    for (int i = 0; i < collection.size(); i++) {
        int counter = 0;

        for (int j = 0; j < collection.size(); j++) {
            if (collection[i] == collection[j]) {
                counter++;
            }
        }

        if (counter > result) {
            result = counter;
        }
    }

    return result;
}

File: python/Find Count of Most Frequent Item in an Array.py
def most_frequent_item_count(collection):
    result = 0

    for i in range(len(collection)):
        counter = 0

        for j in range(len(collection)):
            if collection[i] == collection[j]:
                counter += 1

        if counter > result:
            result = counter

    return result